frontend/demo/src/pages/Inbox/Messenger/MessengerListItem/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import _redux from 'redux';
+import _, {connect, ConnectedProps} from 'react-redux';
+import {Message, MessageSenderType} from '../../../../model/Message';
+import {StateModel} from '../../../../reducers';
+import Avatar from '../MessageList/Avatar';
+
+import styles from './index.module.scss';
+
+type MessengerListItemProps = {
+  messageText: string;
+  messageSenderType: string;
+  messageDate: Date;
+  message: Message;
+} & ConnectedProps<typeof connector>;
+
+const mapStateToProps = (state: StateModel) => {
+  return {
+    lastMessages: state.data.conversations.all.items,
+  };
+};
+
+const connector = connect(mapStateToProps, null);
+
+const MessengerListItem = (props: MessengerListItemProps) => {
+  const {messageText, messageSenderType, message, lastMessages} = props;
+  const isUser = messageSenderType !== MessageSenderType.appUser;
+
+  const messageAvatar = (messageId: string) => {
+    Object.values(lastMessages).forEach(lastMessage => {
+      if (messageId === lastMessage.lastMessage.id) {
+        return <Avatar isLastMessage={true} avatarUrl={lastMessage.contact.avatarUrl} />;
```

Comment:
Out of curiosity couldn't you inline the if statement to the `isLastMessage` and remove the need for two returns and the else?